app/views/podcasts/show.rss.builder

```diff
@@ -32,10 +32,10 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
     xml.image do
       xml.url @feed_image.url
       xml.title @feed.title || @podcast.title
-      xml.link @feed_image.link || @podcast.link
+      xml.link @podcast.link
```

Comment:
[The docs](https://www.rssboard.org/rss-specification#ltimagegtSubelementOfLtchannelgt) indicate:

> Note, in practice the image <title> and <link> should have the same value as the channel's <title> and <link>

So I ended up dropping `link` from all the images (they were all null anyways).